readme.md

```diff
@@ -14,6 +14,542 @@ Swift AWS Lambda Runtime was designed to make building Lambda functions in Swift
 
 If you have never used AWS Lambda or Docker before, check out this [getting started guide](https://fabianfett.de/getting-started-with-swift-aws-lambda-runtime) which helps you with every step from zero to a running Lambda.
 
+First, create a SwiftPM project and pull Swift AWS Lambda Runtime as dependency into your project
+
+ ```swift
+ // swift-tools-version:5.7
+
+ import PackageDescription
+
+ let package = Package(
+     name: "MyLambda",
+     products: [
+         .executable(name: "MyLambda", targets: ["MyLambda"]),
+     ],
+     dependencies: [
+         .package(url: "https://github.com/swift-server/swift-aws-lambda-runtime.git", from: "1.0.0-alpha"),
+     ],
+     targets: [
+         .executableTarget(name: "MyLambda", dependencies: [
+           .product(name: "AWSLambdaRuntime", package: "swift-aws-lambda-runtime"),
+         ]),
+     ]
+ )
+ ```
+
+Next, create a `MyLambda.swift` and implement your Lambda.
+
+### Using async function
+
+ The simplest way to use `AWSLambdaRuntime` is to use the `SimpleLambdaHandler` protocol and pass in an async function, for example:
+
+ ```swift
+ // Import the module
+ import AWSLambdaRuntime
+
+ @main
+ struct MyLambda: SimpleLambdaHandler {
+     // in this example we are receiving and responding with strings
+     func handle(_ name: String, context: LambdaContext) async throws -> String {
+         "Hello, \(name)"
+     }
+ }
+ ```
+
+ More commonly, the event would be a JSON, which is modeled using `Codable`, for example:
+
+ ```swift
+ // Import the module
+ import AWSLambdaRuntime
+
+ // Request, uses Codable for transparent JSON encoding
+ struct Request: Codable {
+   let name: String
+ }
+
+ // Response, uses Codable for transparent JSON encoding
+ struct Response: Codable {
+   let message: String
+ }
+
+ @main
+ struct MyLambda: SimpleLambdaHandler {
+     // In this example we are receiving and responding with `Codable`.
+     func handle(_ request: Request, context: LambdaContext) async throws -> Response {
+         Response(message: "Hello, \(request.name)")
+     }
+ }
+ ```
+
+ Since most Lambda functions are triggered by events originating in the AWS platform like `SNS`, `SQS` or `APIGateway`, the [Swift AWS Lambda Events](http://github.com/swift-server/swift-aws-lambda-events) package includes an `AWSLambdaEvents` module that provides implementations for most common AWS event types further simplifying writing Lambda functions. For example, handling an `SQS` message:
```

Comment:
```suggestion
 Since most Lambda functions are triggered by events originating in the AWS platform like `SNS`, `SQS` or `APIGateway`, the [Swift AWS Lambda Events](http://github.com/swift-server/swift-aws-lambda-events) package includes an `AWSLambdaEvents` module that provides implementations for most common AWS event types further simplifying writing Lambda functions. For example, handling a `SQS` message:
```